tests/test_redshift.py

```diff
@@ -379,17 +380,18 @@ def test_redshift_snapshot_unmark(self):
                     {
                         "type": "value",
                         "key": "SnapshotIdentifier",
-                        "value": "c7n-snapshot",
+                        "value": "c7n-test-snapshot",
                     }
                 ],
                 "actions": [{"type": "unmark"}],
             },
-            session_factory=factory,
+            session_factory=factory, config={'account_id': '644160558196'}
         )
 
         resources = p.run()
         self.assertEqual(len(resources), 1)
-        arn = p.resource_manager.generate_arn(resources[0]["SnapshotIdentifier"])
+        arn = p.resource_manager.generate_arn(
```

Comment:
use get_arns which is the public interface, to avoid manually constructing the identifer.

Reply:
Good call, addressed!